src/features/files/components/LibraryImage.tsx

```diff
@@ -1,27 +1,51 @@
-import { FC } from 'react';
+import { FC, useRef } from 'react';
 import Image from 'next/image';
 
 import TransparentGridBackground from './TransparentGridBackground';
 import { ZetkinFile } from 'utils/types/zetkin';
 
 interface LibraryImageProps {
   imageFile: ZetkinFile;
-  onLoad?: () => void;
-  onLoadingComplete?: () => void;
+  onLoad?: ({ height, width }: { height: number; width: number }) => void;
```

Comment:
Why does this function need the height/width parameters? If I understand the code correctly, this function is called when the loading starts, at which point the values will always be 0/0.

Reply:
As I read the [documentation](https://nextjs.org/docs/pages/api-reference/components/image#onloadingcomplete), `onLoadingComplete` is deprecated with `onLoad` as replacement, which is also called when the image has been loaded. I think this also means `LibraryImageCard` use of onLoad should be changed, is this PR a okay place to do this update too?

Reply:
I think I'd prefer to remove onLoadingComplete and only use onLoad as it give the img as a argument, so there's no need for the `ref` stuff

Reply:
HUGE improvement. I like it!

![Make it so](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExZmV5Z2VqZXhobmY2ZjlmczN5dGE3N3JqbzNzOHBydjZpYThrMnU5NiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/wNlks0ID1igO4/giphy.gif)